### ISSUE TITLE
gofumpt: apparent false positive for certain comments

### ISSUE THREAD

**Issue by flimzy** [golangci/golangci-lint#3135]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).
- [X] Yes, I've tried with the standalone linter if available. (https://golangci-lint.run/usage/linters/)

### Description of the problem

With 1.48.0 and 1.49.0, I see the below failure. With 1.47.3 and earlier I do not.  I don't see any `gofumpt` related changes since 1.46.0, so I don't think the problem is with the linter itself, but I have only browsed the changelog, and not looked deeply into the commit history.

```
main.go:6: File is not `gofumpt`-ed (gofumpt)
```

Although `gofumpt -d` shows no changes.

I'm using the latest version of gofumpt, as well:

```
$ gofumpt --version
v0.3.1
```

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
golangci-lint has version 1.48.0 built from 2d8fea81 on 2022-08-04T18:44:38Z
```
Also 1.49.0
</details>

### Configuration file

<details>

```console
$ cat .golangci.toml
$ cat .golangci.toml 
[linters]
disable-all = true
enable = [
  "gofumpt"
]
```

</details>

### Go environment

<details>

```console
$ go version && go env
go version go1.18.1 linux/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/jonhall/.cache/go-build"
GOENV="/home/jonhall/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/jonhall/go/pkg/mod"
GONOPROXY="gitlab.com/FlashbackSRS/priv,github.com/invopop,github.com/hubuc"
GONOSUMDB="gitlab.com/FlashbackSRS/priv,github.com/invopop,github.com/hubuc"
GOOS="linux"
GOPATH="/home/jonhall/go"
GOPRIVATE="gitlab.com/FlashbackSRS/priv,github.com/invopop,github.com/hubuc"
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.18.1"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/jonhall/src/x/go.mod"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build246500773=/tmp/go-build -gno-record-gcc-switches"

```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
INFO [config_reader] Config search paths: [./ /home/jonhall/src/x /home/jonhall/src /home/jonhall /home /] 
INFO [config_reader] Used config file .golangci.toml 
INFO [lintersdb] Active 1 linters: [gofumpt]      
INFO [loader] Go packages loading at mode 7 (files|compiled_files|name) took 41.67844ms 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 60.903µs 
INFO [linters context/goanalysis] analyzers took 163.435µs with top 10 stages: gofumpt: 163.435µs 
INFO [runner] Processors filtering stat (out/in): max_per_file_from_linter: 1/1, source_code: 1/1, cgo: 1/1, autogenerated_exclude: 1/1, uniq_by_line: 1/1, max_from_linter: 1/1, path_shortener: 1/1, path_prettifier: 1/1, exclude: 1/1, nolint: 1/1, sort_results: 1/1, filename_unadjuster: 1/1, diff: 1/1, path_prefixer: 1/1, exclude-rules: 1/1, max_same_issues: 1/1, severity-rules: 1/1, skip_files: 1/1, skip_dirs: 1/1, identifier_marker: 1/1 
INFO [runner] processing took 109.082µs with stages: autogenerated_exclude: 26.419µs, nolint: 22.451µs, identifier_marker: 20.378µs, source_code: 11.521µs, path_prettifier: 9.848µs, exclude-rules: 8.547µs, skip_dirs: 4.199µs, max_same_issues: 991ns, uniq_by_line: 952ns, cgo: 711ns, path_shortener: 621ns, max_from_linter: 581ns, max_per_file_from_linter: 451ns, filename_unadjuster: 441ns, exclude: 280ns, skip_files: 180ns, sort_results: 180ns, severity-rules: 141ns, diff: 100ns, path_prefixer: 90ns 
INFO [runner] linters took 16.596492ms with stages: gofumpt: 16.427817ms 
main.go:6: File is not `gofumpt`-ed (gofumpt)
// - Accounts.GetFundingAccount()    -- if we're called from a ledger service API like InternalTools.ReTriggerSubLedgerCreation()
// - Accounts.ListFundingAccounts()  -- if walletID is nil, we call this before doing CreateFundingAccount()
//                                      to know the number of accounts before/after, also on RPC timeout we call it again,
//                                      in case the account was created but the call still timed out
// - Accounts.CreateFundingAccount() -- actually creating the funding account on SEFIDE
// - Accounts.CreateIBAN()           -- if there isn't already an IBAN account on the SEFIDE funding account
// - Accounts.DeleteFundingAccount() -- to remove the temporary ("temporal") internal funding account SEFIDE creates
INFO File cache stats: 1 entries of total size 1.1KiB 
INFO Memory: 2 samples, avg is 54.8MB, max is 54.8MB 
INFO Execution took 62.687754ms                   

```

</details>

### Code example or link to a public repository

<details>

```go
package main

// main returns a models.Ledger representing a newly created funding account for SEFIDEsome RPCs.
//
// The method calls, directly or indirectly, our sefide service's APIs:
// - Accounts.GetFundingAccount()    -- if we're called from a ledger service API like InternalTools.ReTriggerSubLedgerCreation()
// - Accounts.ListFundingAccounts()  -- if walletID is nil, we call this before doing CreateFundingAccount()
//                                      to know the number of accounts before/after, also on RPC timeout we call it again,
//                                      in case the account was created but the call still timed out
// - Accounts.CreateFundingAccount() -- actually creating the funding account on SEFIDE
// - Accounts.CreateIBAN()           -- if there isn't already an IBAN account on the SEFIDE funding account
// - Accounts.DeleteFundingAccount() -- to remove the temporary ("temporal") internal funding account SEFIDE creates
//
// We also call the following API on iban-checker service:
// - IbanCheckerClient.Get()         -- to validate the IBAN number
func main() {
}
```

</details>

**Comment by ldez** [comment]
Hello,

it's related to a change in Go1.19 about comment format https://tip.golang.org/doc/go1.19#go-doc

**Comment by flimzy** [comment]
Oh, I should have checked that. Thanks. If that means this should be closed, that's fine.
